index.js

```diff
@@ -1,9 +1,5 @@
 'use strict'
 
 module.exports = {
-  name: 'ember-animated-status-label',
-
-  included(app) {
-    app.import('vendor/animated-status-label.css')
-  }
```

Comment:
There was actually a nasty bug lurking here that started breaking things as soon as I started updating ember.  We should be calling `this._super.included.apply(this, arguments)` somewhere in here.

Rather than add that line, I removed the explicit importing of css.  The addon will automatically import this css into the app if we stick it under `addon/styles`.